src/middlewares/validateInvite.js

```diff
@@ -10,11 +10,11 @@ import { findSingleInvite } from '../services/inviteServices';
  */
 export const validateInvite = async (req, res, next) => {
   const { inviteId } = req.params;
-  if (!inviteId) {
+  if (!inviteId && !isNaN(inviteId)) {
```

Comment:
`inviteId` is a UUID type not an integer.  Validate UUID by creating a validateUUID.js file and put below code in it. Then put inside your route

```/**
 * validate UUID
 * @param {Object} req
 * @param {Object} res
 * @param {Function} next
 * @returns {Object} error
 */
export const validateUUID = (req, res, next) => {
  const validUUID = /^[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}$/;
  const commentSchema = Joi.object().keys({
    inviteId: Joi.string().required()
      .regex(validUUID).error(() => ({ message: 'Enter a valid id' })),
  });

  const errors = joiValidator(req.body, commentSchema);

  if (!errors) {
    return next();
  }
  return respondWithWarning(res, 400, 'Bad Input', errors);
};
```